tests/integration/test_item.py

```diff
@@ -175,6 +175,12 @@ def test_public_token():
             create_response['public_token'])
         assert exchange_response['access_token'] is not None
 
+def test_sandbox_public_token():
+    client = create_client()
+    create_response = client.Sandbox.public_token.create(
+        SANDBOX_INSTITUTION, ['transactions'])
+    assert create_response['access_token'] is not None
```

Comment:
This should be `public_token` -- looks like CI isn't running against PRs, I will look into that

Reply:
my bad, fixed!